uizacoresdk/src/main/res/layout/layout_stats_for_nerds.xml

```diff
@@ -2,14 +2,12 @@
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    >
+    android:layout_height="wrap_content">
```

Comment:
Please revert this change if you don't really change it!

Reply:
it's OK Nam, this changes make the view easier to read.